src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,14 @@ Keywords:
 
 # Release notes
 
+## January 25, 2024
+
+### AEP React Native Edge 5.1.0
```

Comment:
can we remove AEP prefix since it is implied for all libraries and rename the React Native releases like this: `React Native Edge library 5.1.0`

Reply:
cc @cacheung @yangyansong-adbe

Reply:
@yangyansong-adbe How do we make the requested change? Create a PR?

Reply:
I will manually update this PR and update the automation script for generating future PRs.

Reply:
@emdobrin  https://github.com/AdobeDocs/aep-mobile-sdkdocs/pull/487/files#diff-4b2c7af45d3eaa934e113b20a4cda4fb10d3248d72d1055e25873705c4629cddR12